messages/ruby/lib/cucumber/messages.dtos.rb

```diff
@@ -18,10 +23,8 @@ module Messages
     #  It is not to be used for runtime errors raised/thrown during execution. This
     #  is captured in `TestResult`.
     #
-    # The code was auto-generated by {this script}[https://github.com/cucumber/common/blob/main/messages/jsonschema/scripts/codegen.rb]
-    #
 
-    class Attachment
+    class Attachment < ::Cucumber::Messages::Message
```

Comment:
Stylistic question, flagging here.

Is there a reason we're not opting for the 1 class per file structure (I know this is auto-genned, but it still should be possible / desirable)

Reply:
The codegen script has been built like that, it is easier to remain consistent with other generated languages like typescript, and generating a single file for messages is really easier at the moment.